src/_data/capability-matrix.yml

```diff
@@ -202,6 +202,32 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ''
+      - name: Splittable DoFn
+        values:
+          - class: model
+            l1: 'Partially'
+            l2: DoFn with non-monolithic element processing
+            l3: Allows users to develop DoFn's that process a single element in portions ("restrictions"), executed in parallel or sequentially, making it possible to use DoFn for developing bounded and unbounded data sources much more flexible and composable with the rest of the Beam model than the Source API. See http://s.apache.org/splittable-do-fn. Design is in progress on achieving parity with Source API regarding progress signals.
```

Comment:
I think keeping it more descriptive and pithy rather than just relating to the `Source API`. Maybe

```
Allows users to develop DoFn's that process a single element in portions ("restrictions"), executed in parallel or sequentially. This supersedes the unbounded and bounded `Source` APIs by supporting all of their features on a per-element basis. See http://s.apache.org/splittable-do-fn. Design is in progress on achieving parity with Source API regarding progress signals.
```